### PR TITLE
Blog - Using Docker Init in Go

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@adamgordonbell [Preview link](https://6578389418523f46b4905287--ephemeral-narwhal-3ac3b2.netlify.app/blog/docker-init-in-go/)

File: blog/_posts/2023-12-18-docker-init-in-go.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Before Docker Init came into existence, turning a Go project into a containerize
 
 For a quick test setup, you could get away with writing a basic Dockerfile, like so:
 
-~~~{.dockerfile caption=""}
+~~~{.dockerfile caption="Dockerfile"}
```
Comment (adamgordonbell) [inline blog/_posts/2023-12-18-docker-init-in-go.md:37]: @msali123  I had to add these file names.

File: blog/_posts/2023-12-18-docker-init-in-go.md [hunk 2]
```diff
@@ -212,7 +212,7 @@ In the `build` stage, Docker uses the `golang` image to download all dependencie
 
 In addition, `docker init` generously adds comments to all the steps and actions so that even Docker newcomers can quickly navigate their way through the build steps:
 
-~~~{ caption=""}
+~~~{.dockerfile caption="Dockerfile"}
```
Comment (adamgordonbell) [inline blog/_posts/2023-12-18-docker-init-in-go.md:215]: Similar, missing file name

File: blog/_posts/2023-12-18-docker-init-in-go.md [hunk 3]
```diff
@@ -351,7 +351,7 @@ The third file that `docker init` generates is a `.dockerignore` file with a sen
 
 If your Go build process is more complex than a simple `go build`, you'll want to review the output for artifacts that should stay out of the image and add them here:
 
-~~~{ caption=""}
+~~~{ caption=".dockerignore"}
```
Comment (adamgordonbell) [inline blog/_posts/2023-12-18-docker-init-in-go.md:354]: Similar, missing file name

**Review by adamgordonbell — APPROVED** [review]

**Pull request merged**